spark_matcher/utils.py

```diff
@@ -3,14 +3,35 @@
 #          Frits Hermans
 
 from typing import List
-
+import logging
 import numpy as np
 import pandas as pd
 from pyspark.ml.feature import StopWordsRemover
 from pyspark.sql import DataFrame
 from pyspark.sql import functions as F
 
 
+def create_logger() -> logging.Logger:
+    """
+        Creates a logger
+    """
+    logger = logging.getLogger('debug_spark_matcher')
+    logger.setLevel(logging.DEBUG)
```

Comment:
i would propose to make the default logging level 'INFO', then all the cached tables are not shown by default